install/kubernetes/cilium/templates/cilium-agent-daemonset.yaml

```
@@ -370,6 +372,16 @@ spec:
 {{- end }}
       hostNetwork: true
       initContainers:
+        {{/* Required to mount cgroup v2 filesystem on the underlying Kubernetes node.*/}}
```

If using a template comment, please [add spaces](https://helm.sh/docs/chart_best_practices/templates/#formatting-templates) after `{{` and before `}}`.
```suggestion
      {{ /* Required to mount cgroup v2 filesystem on the underlying Kubernetes node */ }}
```

But should this just be a [YAML comment instead](https://helm.sh/docs/chart_best_practices/templates/#comments-yaml-comments-vs-template-comments)? It doesn't document something specific to the template?